StripePayments/StripePayments/Source/PaymentHandler/STPPaymentHandler.swift

```diff
@@ -1494,7 +1494,7 @@ public class STPPaymentHandler: NSObject {
                                         // If this is a web-based 3DS2 transaction that is still in requires_action, we may just need to refresh the PI a few more times.
                                         // Also retry a few times for app redirects, the redirect flow is fast and sometimes the intent doesn't update quick enough
                                         let shouldRetryForCard = retrievedPaymentIntent?.paymentMethod?.type == .card && retrievedPaymentIntent?.nextAction?.type == .useStripeSDK
-                                        let shouldRetryForAppRedirect = [.cashApp, .swish].contains(retrievedPaymentIntent?.paymentMethod?.type)
```

Comment:
I don't know why we were polling for Swish, it is app-to-app but uses the redirect trampoline, so I removed it from polling PMs.

Reply:
nIt: feels like this should be a separate PR, but i'm fine with leaving this in -- i'll leave it up to you if you want to put in a different commit.

Reply:
Yeah agree, typically wouldn't include this in here but since I refactored it a bit I decided why not 🤷